src/runtime/Makefile

```diff
@@ -329,6 +331,18 @@ ifneq (,$(QEMUCMD))
 
     CONFIGS += $(CONFIG_QEMU_SEV)
 
+	CONFIG_FILE_REMOTE = configuration-remote.toml
```

Comment:
nit: format

Reply:
I've switched the tab for spaces, which I hope was the ask?